federation/pkg/federation-controller/service/servicecontroller.go

```diff
@@ -67,6 +67,8 @@ const (
 	UserAgentName = "federation-service-controller"
 	KubeAPIQPS    = 20.0
 	KubeAPIBurst  = 30
+
+	maxNoOfClusters = 256
```

Comment:
We have informally talked about this before but we have never established this formally. We plan to support 100 clusters initially. 256 seems too high to start with.